src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -247,46 +247,46 @@ public boolean isFlattened(Field f) {
      * @param offset indication of where the variable resides in a Java heap
      *        object, if any, else a memory address locating the variable
      *        statically
-     * @param vc inline class
+     * @param pc primitive class
      * @param v the value to store into the indicated Java variable
      * @param <V> the type of a value
      * @throws RuntimeException No defined exceptions are thrown, not even
      *         {@link NullPointerException}
      */
     @IntrinsicCandidate
-    public native <V> void putValue(Object o, long offset, Class<?> vc, V v);
+    public native <V> void putValue(Object o, long offset, Class<?> pc, V v);
 
     /**
-     * Fetches a reference value of type {@code vc} from a given Java variable.
+     * Fetches a reference value of type {@code pc} from a given Java variable.
      * This method can return a reference to a value or a null reference
-     * for a nullable-projection of an inline type.
+     * for a nullable reference of a primitive type.
      *
-     * @param vc inline class
+     * @param pc primitive class
      */
-    public Object getReference(Object o, long offset, Class<?> vc) {
+    public Object getReference(Object o, long offset, Class<?> pc) {
         Object ref = getReference(o, offset);
-        if (ref == null && vc.isInlineClass()) {
-            // If the type of the returned reference is a regular inline type
+        if (ref == null && pc.isPrimitiveClass()) {
+            // If the type of the returned reference is a regular primitive type
             // return an uninitialized default value if null
-            ref = uninitializedDefaultValue(vc);
+            ref = uninitializedDefaultValue(pc);
         }
         return ref;
     }
 
     public Object getReferenceVolatile(Object o, long offset, Class<?> vc) {
```

Comment:
`vc` parameter should also be renamed to `pc` to be consistent with other methods.